src/main/resources/application.conf

```diff
@@ -43,9 +61,9 @@ Tablestore = {
     // link, `user` and `password`.
     // AI2 Internal users: use the database in the `test` deploy environment and the same username. You can get the password
     // from this file in the ops-keystore in S3.
-    url = "jdbc:postgresql://"${POSTGRES_DB}":5432/okcorpus"
+    url = "jdbc:postgresql://okcorpus-dev.civm9kvdku8h.us-west-2.rds.amazonaws.com:5432/okcorpus"
     user = "okcorpus"
-    password = ${POSTGRES_PASSWORD}
+    password = "InteractiveKEtest"
```

Comment:
I don't think we can have passwords and hostnames in here. In fact, you might have to change the password now. Isn't IKE open source? You should change it immediately!